seeders/DatabaseSeeder.ts

```diff
@@ -51,7 +51,7 @@ export class DatabaseSeeder extends Seeder {
 
     const user = new User({
       externalId: USER_EXTERNAL_ID,
-      name: 'Test user',
+      name: process.env.SEEDER_USER_NAME ?? 'Test user',
```

Comment:
Is it required? We have a lot of dummy values in the seeder. We can maybe add some config JSON with all values.

Reply:
Yes, it's important for the user name in bee-stack to be configurable, because now it shows "Hi Test"
![image](https://github.com/user-attachments/assets/f5876594-b041-4bf9-aa60-162368ca9fd1)

Reply:
Can't we just use a better name but keep it hardcoded? Like "Anon" :)

Reply:
Also how come it shows `Test` when the name is `Test user`? Does the UI only take the first word?

Reply:
I prefer env variables instead of having to deal with files/configmaps in docker compose / kubernetes or both options, not just config :)

Reply:
Did you test this? I think that the name comes from the JWT token.